charts/camunda-platform-8.2/test/integration/scenarios/chart-full-setup/values-integration-test-ingress.yaml

```diff
@@ -94,3 +94,10 @@ console:
 
 elasticsearch:
   maxUnavailable: 0
+
+keycloak:
+  service:
+    ports:
+      http: 80
+      https: 443
```

Comment:
Lets take this...

Reply:
and move it here:
https://github.com/camunda/camunda-platform-helm/pull/2893/files#diff-67055a7607e2fab3dacadb7bbe03b5798f07528e6ea9d36d47086353c3128806R44

Reply:
i added commit for this